http/security/apikey/authorizer.go

```diff
@@ -38,6 +38,9 @@ func (b *token) GetValue() string {
 
 // NewAuthorizer returns a new Authorizer for api key authorization with a config and a valid api key.
 func NewAuthorizer(authConfig *Config, apiKey string) *Authorizer {
+	if authConfig.Name == "Authorization" {
+		log.Warnf("The ProfileKey Authorization Config contains 'Authorization' as Header, this can can cause problems: %v ", authConfig)
```

Comment:
There are two problems with this log line. Firstly we don't want to create warnings too lightly, because they will cause a warning in the system observing the log, which in turn will notify a human who has to look at this. So this would happen every time a service using this is started. A debug log would be enough, because if there is a problem somebody will look into the log and see the debug output. A info log would be reasonable, too, I guess. But as this isn't necessarily a misconfiguration, I think the debug level would be better. But then there is the other problem with this: you are using the default logger. But we generally only use the logger that is in the current context (`log.Ctx(ctx)`), because that gives the service the control about how logging is done. If a service would use a different logger that for example sends the logs to a log server, by using the default logger you would bypass this and the log would not end up where it should be. So the solution is to pass the context into this function. But that seems unreasonable for something that is already documented elsewhere. So maybe just drop the log line.

Reply:
It seems I was wrong about the second part. Using `log.Ctx(ctx)` is not what we generally do in this project, it seems. Just leaving this here as a note.